pyanaconda/flags.py

```diff
@@ -62,6 +62,7 @@ def __init__(self, read_cmdline=True):
         self.askmethod = False
         self.eject = True
         self.extlinux = False
+        self.blscfg = True
```

Comment:
This flag should be set in `anaconda.py`, otherwise it will be always `True`.
See: https://github.com/rhinstaller/anaconda/blob/master/anaconda.py#L444

Reply:
Thanks for debugging this; I've fixed it in this PR now.